job-dsls/jobs/pr_jobs.groovy

```diff
@@ -216,13 +216,14 @@ for (repoConfig in REPO_CONFIGS) {
 
         triggers {
             githubPullRequest {
+                useGitHubHooks()
                 orgWhitelist(["kiegroup"])
+                userWhitelist(["kiegroup"])
```

Comment:
userWhiteList? I thought that orgWhitelist is enough?

Reply:
removed